### PR TITLE
Update intro-to-browser-pro-features.mdx

### PR DESCRIPTION
**Pull request by ghost**
Addresses feedback from https://issues.newrelic.com/browse/NR-98785. Since these features are available to both browser Pro and Pro+SPA (and mentions the limitations of Lite on https://docs.newrelic.com/src/content/docs/browser/new-relic-browser/configuration/change-browser-agent-type#agent-types), I removed any mention of which features you get with which agent type. 

I changed one aspect of the page path that included /browser-pro-reatures/, but the browser team and I have plans to soon integrate all these docs into a singular /browser/ location, but that'll be in a future PR. 

I also added descriptions to a few of the features to parallel the other pages that already have the descriptions in their page titles.

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+metaDescription: 'The features associated with the New Relic browser monitoring Pro and Pro+SPA agents.'
+redirects:
+  - /docs/browser/new-relic-browser/browser-pro-features
+  - /docs/browser/new-relic-browser/browser-pro-features/intro-to-browser-pro-features
+---
+
+Check out our key browser monitoring features:
+
+* [**Summary**: View a summary of your browser's performance](/docs/browser/browser-monitoring/getting-started/browser-summary-page/)
+* [**Web vitals**: Learn what contributes to core web vitals](/docs/browser/new-relic-browser/guides/guide-to-monitoring-core-web-vitals/)
```
Comment (nbaenam) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx:12]: This link doesn't work for me. I think the right one is [this](https://docs.newrelic.com/docs/journey-performance/guide-to-monitoring-core-web-vitals/).

**Review by nbaenam — DISMISSED** [review]
It looks good to me! I only have a couple of comments ;-)

File: src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [**Page views**: Examine page performance](/docs/browser/new-relic-browser/browser-pro-features/page-views-examine-page-performance/)
+* [**AJAX**: Identify time-consuming calls](/docs/browser/browser-monitoring/browser-pro-features/ajax-page-identify-time-consuming-calls/)
+* [**JavaScript errors**: Detect and analyze errors](/docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-detect-analyze-errors/)
+* [**Browser error profiles**: Find error causes](/docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes/)
+* [**Session traces**: Explore a webpage's life cycle](/docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle/)
+* [**Geography**: View webpage data by user location](/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location/)
+* [**Browsers**: View webpage data by browser type/platform](/docs/browser/new-relic-browser/browser-pro-features/browsers-problem-patterns-type-or-platform/)
```
Comment (ally-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx:19]:
```suggestion
* [**Browsers**: View webpage data by browser type or platform](/docs/browser/new-relic-browser/browser-pro-features/browsers-problem-patterns-type-or-platform/)
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx (hunk 1 above)
Comment (ally-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx:12]:
```suggestion
* [**Web vitals**: Learn what contributes to core web vitals](/docs/journey-performance/guide-to-monitoring-core-web-vitals)
```

**Pull request merged**